### PR TITLE
Add modulemap

### PR DESCRIPTION
**Pull request by dflems**

### CODE REVIEW

File: include/SPTDataLoader/module.modulemap [hunk 1]
```diff
@@ -0,0 +1,19 @@
+module SPTDataLoader {
+    umbrella header "SPTDataLoader.h"
+
+    header "SPTDataLoaderAuthoriser.h"
```
Comment (JensAyton) [inline include/SPTDataLoader/module.modulemap]: Surely listing these as well as the `umbrella header` clause is redundant?

Comment (dflems) [reply include/SPTDataLoader/module.modulemap]: I dunno! I'm going by some examples I've seen elsewhere.

Comment (dflems) [reply include/SPTDataLoader/module.modulemap]: Figured it would be best to be as complete as possible.

Comment (JensAyton) [reply include/SPTDataLoader/module.modulemap]: I really don’t think so, this is effectively specifying headers twice which is at best unnecessary complexity

**Pull request merged**